CODE_REVIEW.md

```diff
@@ -0,0 +1,45 @@
+## Code Review ##
+
+### Issues ###
+
+* BookSearchComponent: When you search for a new book and the search result panel opens, the previous search results are not cleared and are visible.
+
+* BookSearchComponent: There is not much error handling if the bookSearch API call fails - the user would not know if an error occured. This could be remedied by using the BookState attribues 'loaded' and 'error'.
+
+### Points of Improvement ###
+
+* In terms of functionality, I would add a button to clear the search input. As of now, you need to manually backspace the input if you want to clear the field.
+    * I added this feature
+
+* BookSearchComponent: I noticed there is a difference in how we are subscribing to data store changes. In this component, we are:
+
+        this.store.select(getAllBooks).subscribe(books => {
+            this.books = books;
+            });
+    In ReadingListCompnent, we subscribe to the store:
+    
+    ```
+    readingList$ = this.store.select(getReadingList);
+    ```
+
+* Reading List Effects: I can not tell if this subtle difference in RxJs operators has an intended purpose:
+    * I noticed we are using 'exhaustMap' operator when fetching the list but we're using the 'concatMap' operator when we're posting to the list.
```

Comment:
Again, you've noticed a thing but what's the reason? Are these appropriate? What's the difference between `exhaustMap` and `concatMap`? Why would we use one over the other?

Reply:
I added a more detailed description of this issue in code_review.md